src/tests/ftest/launch.py

```diff
@@ -339,9 +339,20 @@ def check_remote_output(task, command):
             for output, o_hosts in output_data:
                 n_set = NodeSet.fromlist(o_hosts)
                 lines = str(output).splitlines()
+                print("There are {} lines of output".format(len(lines)))
                 if len(lines) > 1:
-                    output = "\n      {}".format("\n      ".join(lines))
-                print("    {}: rc={}, output: {}".format(n_set, code, output))
+                    print("    {}: rc={}, output:".format(n_set, code))
+                    for line in lines:
+                        try:
+                            print("      {}".format(line))
+                        except IOError:
+                            time.sleep(5)
+                            print('XXX: IO Error')
+                            print("      {}".format(line))
```

Comment:
What problem is this intended to work around? Some kind of buffering issue on stdout?

Reply:
Yes, that's my theory.

It matches as well, this PR passed and we did see the debug line.  The test runs a bunch of stuff by ssh then batches all it's output so it must just be filling a buffer in Jenkins itself somewhere.

The leap build passed on this commit but I've just pushed another one now which removes the debug print and whitelists some more memory locations, hopefully it'll have passed by tomorrow.